R/plot.R

```diff
@@ -27,11 +27,13 @@ plot_marker <- function(time_fixef, time_rng, fixef_vary, x_range, vcov_vary,
   sds <- sds * qnorm((1 + p) / 2)
   lbs <- mea - sds
   ubs <- mea + sds
-  plot(xs, mea, ylim = range(lbs, ubs), type = "l", bty = "l", xlab = xlab,
-       ylab = ylab, xaxs = "i", ...)
+
+  plot(xs, mea, ylim = range(lbs, ubs), type = "n", bty = "l", xlab = xlab,
+       ylab = ylab, ...)
+  grid()
   polygon(x = c(xs, rev(xs)), y = c(lbs, rev(ubs)), border = NA,
           col = gray(0, .1))
-  grid()
+  points(xs, mea, type = "l")
```

Comment:
Maybe I am slow but why the separate `plot(..., type = "n")` followed by the `points()` call? Could a single `plot(..., type = "l")` call not work?

Reply:
The reason here is to create the plot area first, then adding the grid, and finally the actual plot line – otherwise, the grid is printed on top of the line if we use `type = "l"` in the first `plot()` call.